ModelLibrary/Utility/ExternalWebMethod.cs

```diff
@@ -0,0 +1,204 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Xml.Linq;
+using VAdvantage.DataBase;
+using VAdvantage.Model;
+
+namespace VAdvantage.Utility
+{
+    class ExternalWebMethod
+    {
+        /// <summary>
+        /// Call external web service
+        /// </summary>
+        /// <param name="url">URL of the web service</param>
+        /// <param name="ewsData">External web service data</param>
+        /// <returns>String, response</returns>
+        public string CallWebService(string url, ExtWebServiceData ewsData)
+        {
+            string result;
+            
+            try
+            {
+                // Create a byte array of the data we want to send
+                byte[] byteData = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(ewsData));
+
+                // Create the web request
+                HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
```

Comment:
call close method of httpwebrequest to free up connection and resource by service

Reply:
Sir request to auto closer hoti hai and response ke liye using use kiya hua h.

Reply:
ok